app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
You're splitting the command string into a list of strings. This is a good approach, but it would be more efficient and clear to destructure the command into its components directly. For example, you can use `command, source, destination = command.split()`.